CWE/decisions/Romania.txt

```diff
@@ -29,4 +30,195 @@ political_decisions = {
 			factor = 1
 		}
 	}
-}
\ No newline at end of file
+
+	oil_modernization_completion = {
+		picture = "rom_oil"
+		potential = {
+			AND = {
+				owns = 667
+				owns = 672
+				owns = 665
+			}
+			OR = {
+				has_country_flag = nicolae_ceausescu_flag
+				has_country_flag = rom_manescu
+				has_country_flag = gheorghe_apostol_romania
+				has_country_flag = rom_nicu
+				has_country_flag = rom_zoia
+			}
+			OR = { government = proletarian_dictatorship government = proletarian_dictatorship1 government = socialist_democracy } }
+			has_country_flag = romanian_oil_modernization_start
+			NOT = { has_country_flag = romanian_oil_modernization_complete }
+		}
+		
+		allow = {
+			NOT = { has_country_modifier = economic_malaise }
+			war = no
+			money = 100000
+		}
+		
+		effect = {
+			prestige = 100
+			667 = {
+				trade_goods = petroleum
+				any_pop = {
+					limit = { type = farmers }
+					pop_type = labourers
+				}
+				add_province_modifier = { name = oil_field duration = -1 } 			
+			}
+			672 = {
+				trade_goods = petroleum
+				any_pop = {
+					limit = { type = farmers }
+					pop_type = labourers
+				}
+				add_province_modifier = { name = oil_field duration = -1 } 			
+			}
+			665 = {
+				trade_goods = petroleum
+				any_pop = {
+					limit = { type = farmers }
+					pop_type = labourers
+				}
+				add_province_modifier = { name = oil_field duration = -1 } 		
+			}
+			set_country_flag = romanian_oil_modernization_complete
+			treasury = -75000
+		}
+		ai_will_do = {
+			factor = 1
+		}
+	}
+
+
+
+	oil_modernization_hungarian_region = {
+		picture = "rom_oil"
+		potential = {
+			AND = {
+				owns = 2533
+				owns = 647
+			}
+			OR = {
+				has_country_flag = nicolae_ceausescu_flag
+				has_country_flag = rom_manescu
+				has_country_flag = gheorghe_apostol_romania
+				has_country_flag = rom_nicu
+				has_country_flag = rom_zoia
+			}
+			OR = { government = proletarian_dictatorship government = proletarian_dictatorship1 government = socialist_democracy } }
+			has_country_flag = romanian_oil_modernization_start
+			NOT = { has_country_flag = romanian_oil_modernization_complete_hungarian_region }
+		}
+		
+		allow = {
+			NOT = { has_country_modifier = economic_malaise }
+			war = no
+			money = 25000
+		}
+		
+		effect = {
+			prestige = 50
+			2533 = {
+				trade_goods = natural_gas 
+				any_pop = {
+					limit = { type = farmers }
+					pop_type = labourers
+				}
+				add_province_modifier = { name = oil_field duration = -1 } 
+			}
+			647 = {
+				trade_goods = other_minerals
+				any_pop = {
+					limit = { type = farmers }
+					pop_type = labourers
+				}
+			}
+			set_country_flag = romanian_oil_modernization_complete_hungarian_region
+			treasury = -10000
+		}
+		ai_will_do = {
+			factor = 1
+		}
+	}
+
+
+	project_bucharest = {
```

Comment:
This decision will clash with the generic renovate capital decision so I think it is better removed and done purely via events instead like what domper did for the moscow renovation events

Reply:
Alright, I had tried to copy that a little. Do you mean making this one an event instead? This is mainly to keep the capital from being called 'Romaniania' if the decision is taken, as well as to simulate the idea of 'rebuilding' Bucharest under principles taken from the DPRK.

Reply:
Yes. I think a better way to do this without having overlapping decisions is to fire a event first that sets the Germania flag and modifier for a longer time than it takes for the Bucharest event to fire (to clear the flags and modifier and add the province modifier) so the Bucharest event will fire first before the generic one.

Reply:
Alright, I'm limited for time at the moment so until I can fix it I'm just going to remove it for now.

Reply:
Okay